openslides/core/static/js/core/start.js

```diff
@@ -46,6 +47,27 @@ angular.module('OpenSlidesApp.core.start', [])
                 this.bootup();
             },
         };
+
+        // We need to 'ping' the server with a get request to whoami, because then we can decide,
+        // if the server is down or respond with a 403 (this cannot be differentiated with websockets)
+        autoupdate.registerRetryConnectCallback(function () {
+            return $q(function (resolve) {
```

Comment:
Yes, but imho throwing errors should not be used for flow control your code. So either returning true/false or use the $q wrapper to manually resolve/reject the promise.